source/process/release-process.md

```diff
@@ -240,6 +240,22 @@ The final release is cut. If an urgent and important issue needs to be addressed
     - **Note:** If the release contains a security update, also draft a Mailchimp email blast for the [Security Bulletin mailing list](http://eepurl.com/cAl5Rv)
     - Finalize blog post for mattermost.com, test on mobile view, and set timer for 08:00 PST on the day of release
     - Update [feature list](https://about.mattermost.com/pricing/) on mattermost.com with relevant new features
+    
+If a security fix release is required, run through the following steps:
+
+1. Dev:
+    - Verify all patch fixes are backported
+    - Verify all patch fixes are tested (either via unit tests or RCs)
+    - Verify hashes and GPG signatures are correct, once build is cut
+    - Delete RCs after final version is shipped
+    - Merge changes made to release branch into master
+2. PM:
+    - Update the changelog
+    - Submit GitLab MR to update MM to vXXX
```

Comment:
Maybe `Submit GitLab MR to update Mattermost`

Reply:
Done